orttraining/orttraining/python/training/ortmodule/_training_manager.py

```diff
@@ -163,8 +163,7 @@ def backward(ctx, *grad_outputs):
                 for input_name in self._graph_info.user_input_names:
                     # Append to the results the backward output for each input that required grad
                     if input_name in require_grad_names_set:
-                        results.append(_utils._torch_tensor_from_dl_pack(
-                            backward_outputs.dlpack_at(require_grad_names_index),
+                        results.append(_utils._ortvalue_to_torch_tensor(
```

Comment:
Is this OK for non-eager mode? Is `backward_outputs[require_grad_names_index]` of type `ORTValue` for non-eager mode?

Reply:
the result is not with type ORTValue, it is with type aten::tensor, for both eager mode and non-eager mode.
The previously implementation is convert the output to dlpack at line 167 then invoke "_torch_tensor_from_dlpack" to convert to aten::tensor. For eager mode we don't need to use dlpack as the bridge. the change here is just consolidate them under the same function, there is no input/output type change.